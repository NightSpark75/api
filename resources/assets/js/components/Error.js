import React from "react";
import { Link } from "react-router";

export default class Error extends React.Component{
    render(){
        return(   
            <div className="row">
                <div className="col-md-6 col-md-offset-3">
                    <div className="panel panel-warning">
                        <div className="panel-heading"><h4 className="panel-title">error page</h4></div>
                        <div className="panel-body">
                            <h4>{this.props.params.msg}</h4>
                        </div>
                    </div>
                </div>
            </div>    
        );
    }
}