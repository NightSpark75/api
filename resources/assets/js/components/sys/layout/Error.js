import React from "react"

export default class Error extends React.Component {
  render() {
    return (
      <article class="message is-danger">
        <div class="message-body">
          {this.props.params.msg}
        </div>
      </article>
    )
  }
}