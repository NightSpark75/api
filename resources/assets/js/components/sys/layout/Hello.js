import React from "react";
import { Link } from "react-router";

export default class Hello extends React.Component{
    render(){
        const { location } = this.props;
        const containerStyle = {
            marginTop: "60px"
        };
        return(
            <div>
                <span>Standart D2K API</span>
            </div>
        );
    }
}