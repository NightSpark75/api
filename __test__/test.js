import React from 'react'
import renderer from 'react-test-renderer'

import Error from '../resources/assets/js/components/sys/layout/Error'
import Hello from '../resources/assets/js/components/sys/layout/Hello'
 
describe('component/layout/sys', () => {
  it('error', () => {
    let params = {msg: 'test'}
    const app = renderer.create(<Error params={params} />)
    const json = app.toJSON()
    expect(json).toMatchSnapshot()
  })

  it('hello', () => {
    const app = renderer.create(<Hello />)
    const json = app.toJSON()
    expect(json).toMatchSnapshot()
  })
})