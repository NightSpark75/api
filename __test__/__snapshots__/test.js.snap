// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/layout/sys error 1`] = `
<article
  class="message is-danger"
>
  <div
    class="message-body"
  >
    test
  </div>
</article>
`;

exports[`component/layout/sys hello 1`] = `
<div>
  <section
    class="hero is-medium is-primary is-bold"
  >
    <div
      class="hero-body"
    >
      <div
        class="container"
      >
        <h1
          class="title"
        >
          Primary bold title
        </h1>
        <h2
          class="subtitle"
        >
          Primary bold subtitle
        </h2>
      </div>
    </div>
  </section>
</div>
`;
